### ISSUE TITLE
Add a flag to cmake build to prevent building/including external deps.

### ISSUE THREAD

**Issue by ntzrmtthihu777** [SFML/SFML#1236]
Some people have a sane-ish setup which already includes properly compiled vorbis, openal, etc
and sfml can clobber those libraries when compiled for windows.

Add a flag to prevent this for those with this sort of setup.

**Comment by ntzrmtthihu777** [comment]
Also, I do see SFML_USE_SYSTEM_DEPS, but this does not work, when either TRUE or FALSE, as all those libraries get installed regardless.

**Comment by MarioLiebisch** [comment]
Was about to suggest `SFML_USE_SYSTEM_DEPS`. How about rewording the issue then?
